wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java

```diff
@@ -192,11 +192,8 @@ public static boolean isFailedState(String state) throws Exception
             WuStateNameMap.put("RUNNING", WUState.RUNNING);
     }
 
-    public static WUState translateWUState(String state) throws Exception
+    public static WUState translateWUState(String state)
     {
-        if (WuStateNameMap.size() <= 0)
```

Comment:
I'll add to my test case to make sure the map isn't empty, but throwing doesn't make sense.